PiWeb.Api.Training.OidcAuth/MainWindow.xaml.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Windows;
+using Zeiss.PiWeb.Api.Rest.Common.Client;
+using Zeiss.PiWeb.Api.Rest.Common.Utilities;
+using Zeiss.PiWeb.Api.Rest.HttpClient.Data;
+
+namespace PiWeb.Api.Training.OidcAuth
+{
+	/// <summary>
+	/// Interaction logic for MainWindow.xaml
+	/// </summary>
+	public partial class MainWindow : Window
+	{
+		public MainWindow()
+		{
+			InitializeComponent();
+		}
+
+		private async void ConnectButton_Click(object sender, RoutedEventArgs e)
+		{
+			// get the database id from the text box
+			Guid databaseGuid;

```

Comment:
Die Deklaration von databaseGuid kannst du direkt in der Try-Methode machen, also mittels out var databaseGuid. Da kannst du isDatabaseIdValid auch einfach als var definieren, dann meckert Rider nicht mehr.

Reply:
erledigt